sample/kotlin/src/room/kotlin/com/datadog/android/sample/data/db/LogRoom.kt

```diff
@@ -0,0 +1,25 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.android.sample.data.db
+
+import androidx.room.ColumnInfo
+import androidx.room.Entity
+import androidx.room.PrimaryKey
+import java.util.UUID
+
+@Entity(tableName = DatadogDbContract.Logs.TABLE_NAME)
+data class LogRoom(
+    @PrimaryKey
+    @ColumnInfo(name = "id")
```

Comment:
Why isn't this one defined as a constant ?

Reply:
Oh...I missed that ;)